src/log.rs

```diff
@@ -31,12 +31,12 @@ byond_fn!(fn log_write(path, data, ...rest) {
             // write first line, timestamped
             let mut iter = data.split('\n');
             if let Some(line) = iter.next() {
-                write!(file, "[{}] {}\n", Utc::now().format("%F %T%.3f"), line)?;
+                writeln!(file, "[{}] {}", Utc::now().format("%F %T%.3f"), line)?;
```

Comment:
Im like 99% sure its this way to ensure a CRLF ending on windows and linux writes, but I could be completely misunderstanding something.

Reply:
Interesting, clippy brought it up so we'd need to mark it ignored.

Reply:
Nope.
> On all platforms, the newline is the LINE FEED character (`\n`/`U+000A`) alone (no additional CARRIAGE RETURN (`\r`/`U+000D`).
https://doc.rust-lang.org/std/macro.writeln.html

This makes sense, since I don't think clippy would've done that.

Reply:
> Edit: Also - I hope you mean ensuring LF endings.

No we have all logs in CRLF so admins can open them in notepad without the ending being fucked. The clippy thing is valid though.

Reply:
This never wrote CRLF

Reply:
![WindowsTerminal_2023-02-11T18-36-32](https://user-images.githubusercontent.com/35135081/218289968-41551f62-b475-4e41-819e-3071b5a235a3.png)